yeahcheese/resources/views/events_list.blade.php

```diff
@@ -12,7 +12,13 @@
         <div>
             <h3>{{ $event->title }}</h3>
             <p>掲載期間：{{ $event->release_date }} - {{ $event->end_date }} 枚数：{{ $event->pictures->count() }}／キー：{{ $event->auth_key }}
-                <a href="{{ route('events.update') }}">編集する</a>
+                <a href="{{ route('events.update') }}">編集する</a><br>
+                @foreach($pictures as $picture)
```

Comment:
こうしたら想定どおりになりそう
下記のようにするなら、$pictures は渡さなくてよくなるね

```suggestion
                @foreach($event->pictures as $picture)
```